inginious/frontend/plugins/upcoming_tasks/templates/upcoming.html

```diff
@@ -0,0 +1,9 @@
+{# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
+{# more information about the licensing of this file. #}
+
+{# Recursively print a list of sections and their content #}
+
+{% set level = level|default(3, true) %}
+{% with tasks=render.order(course, tasks_data.keys()) %}
```

Comment:
4. Replace by `sort_by_deadline`. The comment was not posted by github with the review...